### PR TITLE
(RE-6481) Create packer template for Ubuntu 15.10 amd64 AMI

### PR DESCRIPTION
**Pull request by heathseals**
This PR adds a packer template to create an AMI artifact for Ubuntu 15.10 amd64.  It also slightly changes the way we bootstrap and cleanup puppet on ec2 builds.

### CODE REVIEW

File: scripts/bootstrap-puppet.sh [hunk 1]
```diff
@@ -7,48 +7,21 @@ if [[ "${TEMPLATE}" == "fedora-22"* ]]; then
...
-q_puppetmaster_install=n
-q_run_updtvpkg=n
-q_vendor_packages_install=y
-EOF
-
-  ## extract and install PE
-  tar -xzvf pe.tar.gz
-  cd `ls -d puppet*`
-  ./puppet-enterprise-installer -a /tmp/answers
+  wget ${PE_URL}/${PE_AGENT}.tar.gz -O pe.tar.gz
```
Comment (mckern) [inline scripts/bootstrap-puppet.sh]: Please to be quoting your raw strings if you're interpolating variables into them.

```
"${PE_URL}/${PE_AGENT}.tar.gz"
```

</nitpick>

File: scripts/cleanup-puppet.sh [hunk 2]
```diff
@@ -14,10 +14,9 @@ if [[ "${PACKER_BUILD_NAME}" =~ .*-nocm ]]; then
   rm -rf /etc/puppetlabs
 fi
 
-# Run the PE uninstaller on Amazon builders 
-if [[ "${PACKER_BUILDER_TYPE}" =~ amazon-.* ]]; then
-  cd `ls -d puppet*`
-  ./puppet-enterprise-uninstaller -d -p -y 
+# Cleanup if installed from PE tar ball
+if [[ -n "${PE_URL}" ]]; then
```
Comment (mckern) [inline scripts/cleanup-puppet.sh]: In reference to quoting strings, quotes are not strictly required inside `[[`. Isn't it ironic? Don't you think?

**Comment by mckern** [comment]
A couple of nits, nothing blocking. :+1:

**Comment by kbarber** [comment]
Thanks!

**Comment by kurtwall** [comment]
Looks fine here. :+1: Thanks!

**Comment by heathseals** [comment]
@kurtwall @kbarber ami-ddad4dbd

**Pull request merged**